### ISSUE TITLE
About pytorch bug

### ISSUE THREAD

**Issue by CYKorea** [cvlab-yonsei/MNAD#39]
As I know, pytorch ver. 1.1 has a bug which doesn't save the batch normalization parameters, mean & variance.
It yields different results over every evaluation.
Also, at first evaluation, it has adversarial effect due to the unvalid normalization.
I also tried to replicate your result by your own code, but it couldn't generate 88.5(avenue) or 97.0(ped2).
Did you notice the pytorch bug?
How could you acheive the result even with this bug? 
I 'll appreciate your reply. Thank you.
